_guides/managing-oauth-clients.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Managing OAuth Clients
+layout: article
+section: Developing Components
+order: 1
+since: 20190403
+---
+This document reveals the new approach to OAuth utilization within the platform
+and explains how to [manage](#managing-oauth-client) OAuth clients for
+components in a tenant.
+
+## Changes
+
+
+OAuth client was defined for a component in its environment variables. This way
```

Comment:
I'm not sure the user will understand what you are talking about here. I think you should give here a concrete example, say using `Salesforce` component. It's not clear what exactly was defined in environment variables. That's why you should explain what [these two env vars](https://github.com/elasticio/salesforce-component/blob/master/component.json#L7-L11) are for and where you get them from. Then we explain why we need to put them into env vars.